<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Winamp</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/icons/sound_icon-icons.com_74449.ico</normaloff>:/icons/icons/sound_icon-icons.com_74449.ico</iconset>
  </property>
  <widget class="QPushButton" name="pushButtonAdd">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="buttons.qrc">
     <normaloff>:/buttons/buttons/btn_add.png</normaloff>:/buttons/buttons/btn_add.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPrev">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="buttons.qrc">
     <normaloff>:/buttons/buttons/btn_prev.png</normaloff>:/buttons/buttons/btn_prev.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPlay">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="buttons.qrc">
     <normaloff>:/buttons/buttons/btn_play.png</normaloff>:/buttons/buttons/btn_play.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPause">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="buttons.qrc">
     <normaloff>:/buttons/buttons/btn_pause.png</normaloff>:/buttons/buttons/btn_pause.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonStop">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="buttons.qrc">
     <normaloff>:/buttons/buttons/btn_stop.png</normaloff>:/buttons/buttons/btn_stop.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNext">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="buttons.qrc">
     <normaloff>:/buttons/buttons/btn_next.png</normaloff>:/buttons/buttons/btn_next.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="labelCurrentTrack">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>381</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QTableView" name="playlistView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>381</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="buttons.qrc"/>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
